### DISCUSSION TITLE
Home Assistant - Turning LEDs on/off

### DISCUSSION THREAD

**Discussion by empakoso** [awawa-dev/HyperHDR#855] [Show and tell]
Ok, I was looking for a way to turn on the HyperHDR instance on and off using Home Assistant.  Was planning on turning on the LEDs based on presence sensor.  Here are the steps I took if it helps someone out there:

1. Open HASS configuration.yaml file and enter the following (change your IP address to your own HyperHDR device)

######################START of configuration snippet############
shell_command:
  hyperhdr_on: > 
    curl -X POST "http://192.168.0.43:8090/json-rpc?request" -d'{"command":"componentstate","componentstate":{"component":"LEDDEVICE","state": true}}' -H "Content-Type: application/json"
  hyperhdr_off: > 
    curl -X POST "http://192.168.0.43:8090/json-rpc?request" -d'{"command":"componentstate","componentstate":{"component":"LEDDEVICE","state": false}}' -H "Content-Type: application/json"
######################END of configuration snippet############

2. Now restart your HASS.
3. This shell command will now show up under "Developer tools" - "Services" - "Service" (pull down menu)
4. select "Shell Command:hyperhdr_on" and click "Call Service"

Your HyperHDR instance can now be turned on and off by calling the a custom service in HASS

**Comment by jansene** [comment]
A custom HACS integration 'HyperHDR' is available from here: https://github.com/mjoshd/hyperhdr-ha?tab=readme-ov-file#hyperhdr-custom-component-for-home-assistant
